angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -133,8 +140,12 @@ import { ISprkDropdownChoice } from './sprk-dropdown.interfaces';
                 [attr.aria-label]="choice.content.title"
               >
                 <p class="sprk-b-TypeBodyOne">{{ choice.content.title }}</p>
-                <p>{{ choice.content.infoLine1 }}</p>
-                <p>{{ choice.content.infoLine2 }}</p>
+                <p sprkText variant="bodyTwo" *ngIf="choice.content.infoLine1">
```

Comment:
Thank you for adding in the `sprkText` directive here! Good catch! You do need to import that module in the dropdown module file in order for it to work and for angular to apply the CSS class.

 This image shows the current state on the left and the state with the module imported on the right. On the left, without it being imported, Angular does not recognize the directive and does not apply the body two class so the `sprkText` directive that you added isn't doing anything. On the right, I imported it into the dropdown module and now angular recognizes the directive and applies the class. 

![Screen Shot 2021-02-11 at 3 28 51 PM](https://user-images.githubusercontent.com/2117593/107706774-13468580-6c7e-11eb-96c1-0bd0c8777a30.png)

Reply:
This is importing it into the Dropdown module.

![Screen Shot 2021-02-11 at 3 24 48 PM](https://user-images.githubusercontent.com/2117593/107707048-851ecf00-6c7e-11eb-9835-e186d2661f80.png)